--- a/plugins/editor/external/vstgui4/vstgui/lib/vstguifwd.h	2021-09-04 08:04:07.907860321 -0400
+++ b/plugins/editor/external/vstgui4/vstgui/lib/vstguifwd.h	2021-09-04 08:04:29.466129757 -0400
@@ -6,6 +6,7 @@
 
 #include "vstguibase.h"
 #include <functional>
+#include <limits>
 #include <memory>
 
 namespace VSTGUI {
--- a/plugins/vst/external/VST_SDK/VST3_SDK/public.sdk/source/main/moduleinit.h	2021-09-04 08:11:15.510233785 -0400
+++ b/plugins/vst/external/VST_SDK/VST3_SDK/public.sdk/source/main/moduleinit.h	2021-09-04 08:11:18.211267752 -0400
@@ -39,6 +39,7 @@
 #include "pluginterfaces/base/ftypes.h"
 #include <algorithm>
 #include <functional>
+#include <limits>
 #include <numeric>
 
 //------------------------------------------------------------------------
